libraries/abstractions/ble_hal/test/ble_test_scipts/testutils.py

```diff
@@ -137,6 +138,10 @@ def find_gatt_service_in_objects(services, characteristics, descriptors):
         if ifaces.get(SERVICE_INTERFACE) is not None:
             uuid = str(objInterface.Get(SERVICE_INTERFACE, "UUID"))
             primary = objInterface.Get(SERVICE_INTERFACE, "Primary")
+            if ENABLE_TC_AFQP_ADD_INCLUDED_SERVICE == 1:
+                if uuid == "8a7f1168-48af-4efb-83b5-e679f9320001":
+                    included_service = objInterface.Get(SERVICE_INTERFACE, "Includes")
+                    print(included_service)
```

Comment:
Add more information in logs for better debugging. Like print("Included Service: " + str(included_service))

Reply:
sure